src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,77 @@
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type Props = {
+  list: Todo[];
+  activeTodo?: Todo | null | undefined;
```

Comment:
The `activeTodo` prop type includes `undefined`, but since `activeTodo` is already optional (indicated by `?`), it can be `null` or omitted. Including `undefined` is redundant and can be removed for clarity.